words_dir/tweet_adjectives.py

```diff
@@ -453,14 +456,23 @@
     'fast asleep',
 
     # character traits
+    'affable',
+    'agreeable',
+    'amiable',
+    'charming',
     'decadent',
     'dogmatic',
     'fidgety',
+    'gregarious'
     'intimidating',
     'irritable',
+    'likeable',
     'mischievous',
+    'polite',
     'sagacious',
     'stoic',
     'strict',
-    'strong'
+    'strong',
+    'understanding'
+    'weak'
```

Comment:
Would you mind adding a comma to the end of this line?

Reply:
Done